src/templates/banner.js

```diff
@@ -0,0 +1,150 @@
+    setConsentCallback = callback;
+
+    if (!document.getElementById('agttcnsntbnnr')) {
+      bannerParentNode.insertAdjacentHTML('beforeend', bannerContent);
+    }
+
+    consBtnAgree = document.getElementById('consBtnAgree');
+    consBtnDismiss = document.getElementById('consBtnDismiss');
+
+    document.getElementById('agttcnstbnnr').style.display = 'block';
+  }
+
+  function hideConsentBanner() {
+    if (document.getElementById('agttcnstbnnr')) {
+      document.getElementById('agttcnstbnnr').style.display = 'none';
+    }
+  }
+
+  function isConsentBannerVisible() {
+    return !!document.getElementById('agttcnstbnnr') && document.getElementById('agttcnstbnnr').style.display != 'none';
+  }
+
+  function setSelected(element) {
+    if (!element) {
+      return;
+    }
+    element.classList.add('selected');
+    element.style.color = '#ffffff';
+    element.style.backgroundColor = '#76b642';
+  }
+
+  function setNotSelected(element) {
+    if (!element) {
+      return;
+    }
+    element.classList.remove('selected');
+    element.style.color = '#76b642';
+    element.style.backgroundColor = '#ffffff';
+  }
+
+  function handleLeftRight() {
+    if (consBtnAgree && !consBtnAgree.classList.contains('selected')) {
+      setSelected(consBtnAgree);
+      setNotSelected(consBtnDismiss);
+    } else {
+      setSelected(consBtnDismiss);
+      setNotSelected(consBtnAgree);
+    }
+  }
+
+  function handleEnter() {
+    if (consBtnAgree && consBtnAgree.classList.contains('selected')) {
+      setConsentCallback(true);
+    } else {
+      setConsentCallback(false);
+    }
+    hideConsentBanner();
+
+    // reset consent banner to have ACCEPT button selected
+    setSelected(consBtnAgree);
+    setNotSelected(consBtnDismiss);
+  }
+
+  function handleVK(keyCode) {
+    var KeyEvent = window['KeyEvent'] || {};
+    KeyEvent.VK_LEFT = KeyEvent.VK_LEFT || window['VK_LEFT'] || 37;
+    KeyEvent.VK_RIGHT = KeyEvent.VK_RIGHT || window['VK_RIGHT'] || 39;
+    KeyEvent.VK_ENTER = KeyEvent.VK_ENTER || window['VK_ENTER'] || 13;
+
+    switch (keyCode) {
+      case KeyEvent.VK_ENTER:
+        handleEnter();
+        break;
+      case KeyEvent.VK_LEFT:
+      case KeyEvent.VK_RIGHT:
+        handleLeftRight();
+        break;
+      default:
+        break;
+    }
+  }
+
+  var hideBannerTimeout;
+
+  switch (command) {
+    case 'showBanner':
+      showConsentBanner(parameter, callback, 3);
+      hideBannerTimeout = setTimeout(function () {
+        hideConsentBanner();
+        !!callback && callback();
+      }, parseInt('<%-BANNER_TIMEOUT%>'));
+      break;
+    case 'hideBanner':
+      hideConsentBanner();
+      !!callback && callback();
+      break;
+    case 'isBannerVisible':
+      !!callback && callback(isConsentBannerVisible());
+      break;
+    case 'handleKey':
+      if (isConsentBannerVisible()) {
+        var KeyEvent = window['KeyEvent'] || {};
+        KeyEvent.VK_ENTER = KeyEvent.VK_ENTER || window['VK_ENTER'] || 13;
```

Comment:
Why is this `VK_ENTER` config setting here in the case block? This is a constant that does not depend on the call and should be extracted to the outer block (together with the other key configs)